internal/test/stream.go

```diff
@@ -0,0 +1,161 @@
+package test
```

Comment:
Something like `mockstream` may be better package naming.

Reply:
@at-wat what do you think of the name now? Feel free to edit the branch directly!